docs/deployment/dns.md

```diff
@@ -0,0 +1,77 @@
+```yaml
+kind: UDSBundle
+metadata:
+  name: core-with-lb-config
+  description: A UDS example bundle for deploying UDS Core with external Load Balancer configuration
+  version: "0.0.1"
+  
+packages:
+  - name: core
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    ref: 0.27.0-upstream
+
+overrides:
+ istio-admin-gateway:
+   gateway:
+     values:
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-type
+        value: "external"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-scheme
+        value: "internal"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-attributes
+        value: "load_balancing.cross_zone.enabled=true"
+  istio-tenant-gateway:
+    gateway:
+      values:
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-type
+        value: "external"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-scheme
+        value: "internet-facing"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-attributes
+        value: "load_balancing.cross_zone.enabled=true"        
+```
+{{% alert-note %}}
+These service annotations and their values are subject to change. Please reference documentation from your cloud provider to ensure their validity.
+{{% /alert-note %}}
+
+### Istio Gateways
+Once UDS Core is deployed, there will be Istio Gateway resources in your cluster. You can find each Gateway in a dedicated namespace:
+```cli
+$ kubectl get gateway -A
+NAMESPACE              NAME             AGE
+istio-admin-gateway    admin-gateway    1h
+istio-tenant-gateway   tenant-gateway   1h
+```
+
+Each Gateway will have a Kubernetes Service of type Load Balancer:
+```cli
+$ kubectl get svc -A | grep LoadBalancer
+NAMESPACE                   NAME                                             TYPE           CLUSTER-IP      EXTERNAL-IP                                        PORT(S)                                     AGE
+istio-admin-gateway         admin-ingressgateway                             LoadBalancer   10.43.82.84     k8s-istioadm-admin...elb.us-east-1.amazonaws.com   15021:30842/TCP,80:31304/TCP,443:31518/TCP  1h
+istio-tenant-gateway        tenant-ingressgateway                            LoadBalancer   10.43.47.182    k8s-istioten-tenant...elb.us-east-1.amazonaws.com  15021:31222/TCP,80:30456/TCP,443:32508/TCP  1h
+```
+
+From here, DNS records can be created for the environment. You can create A records that associate your top-level domain to the Public IP Addresses of your Load Balancers. For subdomains, CNAME records can be used. For more information on how to configure DNS in AWS, see this [documentation](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-to-elb-load-balancer.html).
```

Comment:
> For more information on how to configure DNS in AWS, see this [documentation](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-to-elb-load-balancer.html).

I think we can leave it if you think it would be valuable, but want to be careful about giving the impression that this only works/has been tested with AWS. Unfortunately I think most docs we could link would be provider specific (i.e. Route53, Cloudflare, ...). Maybe sufficient to just drop the last line and reference "refer to your DNS provider's documentation"?